### PR TITLE
Describe keys for dev_server in generated webpacker.yml

### PR DESCRIPTION
**Pull request by alexeyr-ci1**
Also fix `inline` to `inline_css` in some test files, though it didn't affect results because the default is `inline_css: true`.

### CODE REVIEW

File: lib/install/config/webpacker.yml [hunk 1]
```diff
@@ -70,13 +72,14 @@ development:
...
-      #  hostname: "0.0.0.0"
-      #  pathname: "/ws"
+      #  hostname: '0.0.0.0'
+      #  pathname: '/ws'
       #  port: 8080
     # Should we use gzip compression?
     compress: true
     # Note that apps that do not check the host are vulnerable to DNS rebinding attacks
-    allowed_hosts: "all"
+    allowed_hosts: 'all'
```
Comment (justin808) [inline lib/install/config/webpacker.yml:81]: Why use quotes at all?

Why prefer single?

File: test/mounted_app/test/dummy/config/webpacker.yml [hunk 2]
```diff
@@ -41,8 +41,14 @@ development:
     port: 3035
     public: localhost:3035
     hmr: false
-    # Inline should be set to true if using HMR
-    inline: true
```
Comment (justin808) [inline test/mounted_app/test/dummy/config/webpacker.yml:45]: Why removing? In-line is a different unrelated option?

File: lib/install/config/webpacker.yml [hunk 3]
```diff
@@ -49,11 +49,13 @@ development:
   compiler_strategy: mtime
 
   # Reference: https://webpack.js.org/configuration/dev-server/
+  # Keys not described there are documented inline and in https://github.com/shakacode/shakapacker/
   dev_server:
     https: false
     host: localhost
     port: 3035
     # Hot Module Replacement updates modules while the application is running without a full reload
+    # Used instead of the `hot` key in https://webpack.js.org/configuration/dev-server/#devserverhot
```
Comment (justin808) [inline lib/install/config/webpacker.yml:58]: We can doc other keys here.

File: lib/install/config/webpacker.yml (hunk 1 above)
Comment (alexeyr-ci1) [reply lib/install/config/webpacker.yml:81]: We can remove them, but if we keep them better to have same quotes everywhere (IMHO) and there are more single quotes in this file.

File: test/mounted_app/test/dummy/config/webpacker.yml (hunk 2 above)
Comment (alexeyr-ci1) [reply test/mounted_app/test/dummy/config/webpacker.yml:45]: I've searched and `inline` doesn't seem to be used anywhere, only `inline_css`. Namely, here https://github.com/shakacode/shakapacker/blob/0f688bcfcf4a4d66fd69a59f359fc37241f5f23f/lib/webpacker/dev_server.rb#L59 but there are no `fetch(:inline)` calls.

File: lib/install/config/webpacker.yml (hunk 3 above)
Comment (alexeyr-ci1) [reply lib/install/config/webpacker.yml:58]: My thinking was to document only our own keys, but yes, documenting everything makes sense.

**Review by justin808 — APPROVED** [review]

**Comment by justin808** [comment]
@alexeyr please resolve conflicts and merge.

**Pull request merged**